portfolio/withdraw/withdraw_types.go

```diff
@@ -103,6 +125,10 @@ type Request struct {
 
 	Crypto CryptoRequest `json:"crypto"`
 	Fiat   FiatRequest   `json:"fiat"`
+
+	Travel TravelRule `json:"travel_rule"`
+
```

Comment:
🦍 I get the urge to add space around fields you add.
Feels like you're not touching existing code so much that way.
However I don't think it's helping and it creates staggered indentation.
```suggestion
	Travel TravelRule `json:"travel_rule"`
```

Reply:
It was more attempting to fit into the rest of the structure for the type. It seemed to be separating the fields into blocks of related functionality.

If you think I should collapse all these fields together, I'd be happy to do so.